src/main/java/mezz/jei/util/CommandUtilServer.java

```diff
@@ -157,7 +157,7 @@ private static void giveToInventory(PlayerEntity entityplayermp, ItemStack itemS
 				entityitem.makeFakeItem();
 			}
 
-			entityplayermp.world.playSound(null, entityplayermp.posX, entityplayermp.posY, entityplayermp.posZ, SoundEvents.ENTITY_ITEM_PICKUP, SoundCategory.PLAYERS, 0.2F, ((entityplayermp.getRNG().nextFloat() - entityplayermp.getRNG().nextFloat()) * 0.7F + 1.0F) * 2.0F);
+			entityplayermp.world.playSound(null, entityplayermp.serverPosX, entityplayermp.serverPosY, entityplayermp.serverPosZ, SoundEvents.ENTITY_ITEM_PICKUP, SoundCategory.PLAYERS, 0.2F, ((entityplayermp.getRNG().nextFloat() - entityplayermp.getRNG().nextFloat()) * 0.7F + 1.0F) * 2.0F);
```

Comment:
You missed switching to the getters here.

Reply:
Thanks yep my bad, fixed.